src/model/emergency_class/emergency_class_model.py

```
@@ -0,0 +1,383 @@
+        alert_timestamp=alert_timestamp,
+        order_id=order_id,
+        order_status_id=order_status_id,
+    )
+
+    try:
+        emergency_classification_config = get_default_config()
+        # Check if default config exists
+        if emergency_classification_config is None:
+            raise HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=f"Default emergency classification config (for user with ID 0) not found",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)